bundle/bundle.go

```diff
@@ -343,13 +402,178 @@ func (r *Reader) Read() (Bundle, error) {
 	return bundle, nil
 }
 
+func (r *Reader) checkSignaturesAndDescriptors(signatures *SignaturesConfig, descriptors []*Descriptor) error {
+	if signatures == nil && r.verificationConfig != nil {
+		msg := "bundle reader provided with signature verification config but .signatures.json file not included in the bundle"
+		return fmt.Errorf("%v", msg)
```

Comment:
nit: call `errors.New("...")` or `fmt.Errorf("...")`; not need for the intermediate variable.

Reply:
Done